src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -330,66 +331,62 @@ public async Task ApproveReviewAsync(ClaimsPrincipal user, string reviewId, stri
         /// <summary>
         /// Sends info to AI service for generating initial review on APIReview file
         /// </summary>
-        public async Task<int> GenerateAIReview(string reviewId, string revisionId)
+        public async Task<int> GenerateAIReview(string reviewId, string apiRevisionId)
         {
-            var revisions = await _apiRevisionsManager.GetAPIRevisionsAsync(reviewId);
-            var revision = revisions.Where(r => r.Id == revisionId).FirstOrDefault();
+            var revision = await _apiRevisionsManager.GetAPIRevisionAsync(apiRevisionId: apiRevisionId);
             var codeFile = await _codeFileRepository.GetCodeFileAsync(revision, false);
-            var codeLines = codeFile.RenderText(false);
 
-            var reviewText = new StringBuilder();
-            foreach (var codeLine in codeLines)
-            {
-                reviewText.Append(codeLine.DisplayString);
-                reviewText.Append("\\n");
-            }
-
-            var url = "https://apiview-gpt.azurewebsites.net/python";
+            var codeLines = codeFile.CodeFile.GetApiLines(skipDocs: true);
+            var url = $"https://apiview-gpt.azurewebsites.net/{LanguageServiceHelpers.GetLanguageAliasForCopilotService(revision.Language)}";
             var client = new HttpClient();
             client.Timeout = TimeSpan.FromMinutes(20);
-            var payload = new
+            var payload = new Dictionary<string, object>
             {
-                content = reviewText.ToString()
+                { "target", String.Join("\\n", codeLines.Select(item => item.lineText.Trim())) },
+                { "base", string.Empty },
+                { "diff", string.Empty }
```

Comment:
This can now be removed. We don't need you to send a diff. However, we do need you to send "outline", as discussed in https://github.com/Azure/azure-sdk-tools/issues/10507